app.json

```diff
@@ -19,22 +19,22 @@
     ],
     "ios": {
       "bundleIdentifier": "com.plataformasabia.app",
-      "buildNumber": "1.1.0",
+      "buildNumber": "1.2.0",
       "supportsTablet": false,
       "config": {
-        "googleMapsApiKey": "AIzaSyDlQrq14K2OTjUxioB4fW7NJTzZQ2ZFtxA"
+        "googleMapsApiKey": "AIzaSyBCdMT6B3SyMWcEbZoHuW5Tkbdt_dIC0VU"
       }
     },
     "android": {
       "package": "com.plataformasabia.app",
-      "versionCode": 10,
+      "versionCode": 12,
       "adaptiveIcon": {
         "foregroundImage": "./assets/adaptive-icon.png",
         "backgroundColor": "#00a688"
       },
       "config": {
         "googleMaps": {
-          "apiKey": "AIzaSyDlQrq14K2OTjUxioB4fW7NJTzZQ2ZFtxA"
+          "apiKey": "AIzaSyBCdMT6B3SyMWcEbZoHuW5Tkbdt_dIC0VU"
```

Comment:
We need to move the key to .env... Renew and move... for security reasons.

Reply:
@davioliveiira are already taking care of this, @marcostullyo 🙂